src/daemon/bitcoind/actions.rs

```diff
@@ -581,6 +606,35 @@ fn comprehensive_rescan(
                 }
             }
 
+            // Fourth layer: if the Cancel transaction *only* was confirmed and becomes unconfirmed,
```

Comment:
What do you mean by fourth? I used this layer terminology wrt to the transaction chain:

```
1 - Deposit
2 - Unvault     |      Emergency
3 - Cancel      |      UnvaultEmergency    |     Spend
```

Reply:
Modified it and pushed

Reply:
Ah, now it makes sense! Thanks :)

Reply:
L4 ftw